### PR TITLE
Added cockpit workstation and  kubernetes containers from distro rpms

### PR DESCRIPTION
**Pull request by mohammedzee1000**
These are cockpit containers for cockpit workstation and cockpit kubernetes based on distro rpms.

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
ping @kbsingh @jperrin

File: cockpit/ws/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+FROM registry.centos.org/centos/centos
```
Comment (jperrin) [inline cockpit/ws/Dockerfile]: Do we need to specify the registry here? We should want this to work for folks using their own registries or the docker hub as well.

**Comment by mohammedzee1000** [comment]
@jperrin done

**Comment by jperrin** [comment]
LGTM.

**Pull request merged**